packages/rollup/src/rollup_bundle.bzl

```diff
@@ -0,0 +1,257 @@
+"Rules for running Rollup under Bazel"
+
+load("@build_bazel_rules_nodejs//internal/linker:link_node_modules.bzl", "module_mappings_aspect", "register_node_modules_linker")
+
+_DOC = """Runs the Rollup.js CLI under Bazel.
+
+See https://rollupjs.org/guide/en/#command-line-reference
+
+Typical example:
+```python
+load("@npm_bazel_rollup//:index.bzl", "rollup_bundle")
+
+rollup_bundle(
+    name = "bundle",
+    srcs = ["dependency.js"],
+    entry_points = "input.js",
+    config_file = "rollup.config.js",
+)
+```
+
+Note that the command-line options set by Bazel override what appears in the rollup config file.
+This means that typically a single `rollup.config.js` can contain settings for your whole repo,
+and multiple `rollup_bundle` rules can share the configuration.
+
+Thus, setting options that Bazel controls will have no effect, e.g.
+
+```javascript
+module.exports = {
+    output: { file: 'this_is_ignored.js' },
+}
+```
+
+The rollup_bundle rule always produces a directory output, because it isn't known until
+rollup runs whether the output has many chunks or is a single file.
+
+To get multiple output formats, wrap the rule with a macro or list comprehension, e.g.
+
+```python
+[
+    rollup_bundle(
+        name = "bundle.%s" % format,
+        entry_points = "foo.js",
+        format = format,
+    )
+    for format in [
+        "cjs",
+        "umd",
+    ]
+]
+```
+
+This will produce one output directory per requested format.
+"""
+
+_ROLLUP_ATTRS = {
+    "srcs": attr.label_list(
+        doc = """JavaScript source files from the workspace.
+
+The file(s) passed to entry_points are automatically added.
```

Comment:
I was confused for a second by the phrasing. Maybe "The file(s) passed to entry_points do not need to be specified"?